.travis.yml

```diff
@@ -4,12 +4,16 @@ node_js:
   - "12"
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.21.1
```

Comment:
We could maybe get this from package.json dynamically?

Reply:
Yup sounds good